src/reducers/composers.js

```diff
@@ -97,13 +97,13 @@ export const addDataToMapComposed = (state, action) => {
   // Update mapState store
   mergedState = {
     ...mergedState,
-    mapState: stateMapConfigUpdater(mergedState.mapState, {payload: {mapState: parsedConfig && parsedConfig.mapState}})
+    mapState: stateMapConfigUpdater(parsedConfig.mapState, {payload: {mapState: mergedState.mapState}})
```

Comment:
these are going the wrong way before, we were overwriting what we just calculated with what is originally passed in, causing us to calculate bounds then overwrite them with the previous incorrect bounds.

Reply:
But this is the intended behavior, if you pass mapState in the config, it is always going to override the calculated map bounds. 
https://github.com/uber/kepler.gl/blob/master/README.md#user-content-config-notice

If you want to enable auto center. add {options: {centerMap: true}} to `addDataToMap` and omit passing `mapState` inside `config`

Reply:
These weren't necesarily the wrong way its just we were calculating bounds in the visState updater even though it was in mapState which was then getting overridden when calling `mapStateUpdaters` - `receiveMapConfigUpdater`. I have adjusted the code now so that bounds are calculated when we update mapState instead. This requires passing the visState to the mapStateUpdater however.